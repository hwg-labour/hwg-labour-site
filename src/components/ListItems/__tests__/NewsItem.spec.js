import { NewsItem } from "../index";
import React from "react";
import { shallow } from "enzyme";

const EVENT = {
  id: "82842695-2c07-42ef-aa7d-6170aea375f0",
  image: {
    file: {
      url: "image.png"
    }
  },
  title: "News EVent Title",
  date: Date(),
  membersOnly: false,
  description: "dsjadhiuashdiashidhasd"
};

describe("NewsItem", () => {
  it("renders component", () => {
    const wrapper = shallow(<NewsItem news={EVENT} />);

    expect(wrapper).toMatchSnapshot();
  });
});
