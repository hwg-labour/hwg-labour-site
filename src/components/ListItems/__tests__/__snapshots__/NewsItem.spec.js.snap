// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsItem renders component 1`] = `
<GridRow
  key="82842695-2c07-42ef-aa7d-6170aea375f0-news"
>
  <GridColumn>
    <NewsItem__NewsThumbnail
      as={[Function]}
      src="https://res.cloudinary.com/codogo/image/fetch/w_800,c_fill,g_face,f_auto/https:
					image.png"
      to="/news/news-event-title"
    />
  </GridColumn>
  <GridColumn>
    <GatsbyLink
      to="/news/news-event-title"
    >
      <Header
        as="h3"
      >
        News EVent Title
      </Header>
    </GatsbyLink>
    <p
      style={
        Object {
          "color": "#aaaaaa",
        }
      }
    >
      March 9th 2020
    </p>
    <p />
    <Button
      as={[Function]}
      size="small"
      to="/news/news-event-title"
    >
      Read more 
      <Icon
        as="i"
        name="right arrow"
      />
    </Button>
  </GridColumn>
  <NewsItem__NewsDivider
    section={true}
  />
</GridRow>
`;
